### PR TITLE
Hash type file in human readable fixed

### PR DESCRIPTION
**Pull request by GalRabinDemisto**
## Status
Ready

## Related Issues
https://github.com/demisto/etc/issues/18103

## Description
Bug in the human-readable table, the hash type isn't correct.
I found that the hash type is hardcoded.

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

### CODE REVIEW

**Comment by bakatzir** [comment]
@GalRabinDemisto no need to tag the PR as well.
only the related issue should be tagged.

File: Releases/LatestRelease/integration-AlienVaultOTX.md [hunk 1]
```diff
@@ -0,0 +1 @@
+- bug fix hash file type in human readable.
```
Comment (bakatzir) [inline Releases/LatestRelease/integration-AlienVaultOTX.md]: @GalRabinDemisto RN are in old format, please follow new convention:
https://github.com/demisto/content/tree/master/docs/release_notes

Comment (GalRabinDemisto) [reply Releases/LatestRelease/integration-AlienVaultOTX.md]: Hi,
@bakatzir 
Just change the release notes by the new convention.

Thanks for the review.

**Comment by GalRabinDemisto** [comment]
We decided that we will be fixing it in the rewrite.

**Pull request closed without merging**